docs/pages/serving_an_origin.mdx

```diff
@@ -2,80 +2,195 @@ import ExportedImage from "next-image-export-optimizer";
+  - The certicate file from Let's Encrypt (or another CA), usually named as `example.com.crt` or `example.com.pem`
+  - Default location: `~/.config/pelican/certificates/tls.crt` for non-root users or `/etc/pelican/certificates/tls.crt` for root users.
 
+- `Server.TLSKey`
+  - The private key corresponding to the TLS certificate, usually named as `example.com.key`.
+  - Default location: `~/.config/pelican/certificates/tls.key` for non-root users or `/etc/pelican/certificates/tls.key` for root users.
 
-Grab the appropriate binary for your system from the [pelican repository](https://github.com/PelicanPlatform/pelican/releases). For more details, see [the client installation instructions](/install.mdx)
+- `Server.TLSCACertificateFile`
+  - The intermediate certificate from Let's Encrypt to establish the trust chain to a root certificate, usually named as `letsencrypt-intermediate.pem` or `chain.pem`
+  - Default location: `~/.config/pelican/certificates/tlsca.pem` for non-root users or `/etc/pelican/certificates/tlsca.pem` for root users.
 
-## Serving an Origin
+Since your TLS certificate is associated with your domain name, you will need to change the default hostname of Pelican server to be consistent. Set `Server.Hostname` to your domain name (e.g. `example.com`).
 
-### Register the origin with the namespace-registry
+## Serve an Origin
 
-Before serving an origin, you need to register the origin with the namespace-registry. This can be done by running
+> If you are running Pelican docker image to serve an origin, please refer to the [docker image documentation](./install/docker.mdx#run-pelican-origin-server).
 
-```pelican namespace register --prefix <namespace prefix> --namespace-url <registry url>```
+### Find the federation to join in
 
-Where `<namespace_prefix>` is a namespace prefix associated with the origin and `<registry url>` is the url of the namespace registry. This gives the registry the public key for the origin, and makes that public key available to all of the other services in the federation for verifying tokens
+Before serving an origin, you need to find a Pelican federation to join in. If you are unfamiliar with the term **federation**, refer to [Useful Terminology](./client-usage.mdx#useful-terminology) before proceeding.
 
+If you don't have a federation in mind, Open Science Data Federation (OSDF) is an example Pelican federation that you can join in for testing purposes. If you are interesting in serving an OSDF origin, refer to the [OSDF website](https://osg-htc.org/) for details.
 
-### Running Origin Serve
+The federation discovery URL for OSDF is `osg-htc.org`. You may use this as your `<federation>` argument in the next section when launching your origin.
+
+### Launch the Origin
 
 To launch a pelican origin, run:
 
-```./pelican origin serve -f <federation> -v <local_directory>:<namespace_prefix>```
+```bash
+pelican origin serve -f <federation> -v <local_directory>:<namespace_prefix>
```

Comment:
Agree. I will mention that it's also possible to serve a S3 origin, but that I think should be separated into another PR/even a separate document because I think there will be a bunch of setup and configs to do with a S3 origin.

Reply:
Okay. Let's create an issue for it, then. And probably assign it to Justin.